source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -959,13 +959,6 @@ void ShaderGraph::finalize(GenContext& context)
     // Sort the nodes in topological order.
     topologicalSort();
 
-    // Calculate scopes for all nodes in the graph.
-    //
-    // TODO: Enable calculateScopes() again when support for
-    // conditional nodes are improved.
-    //
-    // calculateScopes();
```

Comment:
This is where the feature where disabled a long time ago by not performing the scope calculations.